onnx/defs/rnn/defs.cc

```diff
@@ -71,6 +90,15 @@ allows to make spec more uniform.
     .Attr("num_layers", "Numbers of RNN layers in the stack, default 1",
           AttrType::INT)
     .Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT)
+    .Attr("activations", "A list of activation functions for the gates. Typical activation "
+          "functions are sigmoid and tanh",
+          AttrType::STRINGS)
+    .Attr("use_bias", "Use the bias tensors for the gates if 1. Default 1.",
+          AttrType::INT)
+    .Attr("reverse", "Process the sequences in reverse order, default 0",
+          AttrType::INT)
+    .Attr("clip", "Cell clip threshold. Default to no clip if not specified.",
```

Comment:
add more comments in the spec above regarding where exactly 'clip' gets plugged in?

Reply:
Done.